packages/babel-preset-flow/src/index.js

```diff
@@ -0,0 +1,9 @@
+import transformSyntaxFlow from "babel-plugin-syntax-flow";
+import transformFlowStripTypes from "babel-plugin-transform-flow-strip-types";
+
+export default {
+  plugins: [
+    transformSyntaxFlow,
```

Comment:
Hmm, I mean it's pretty much a no-op, but I feel like we should remove them, otherwise people reading the presets would think this is necessary.